CoreErrorReportView/src/au/gov/asd/tac/constellation/views/errorreport/ErrorReportTopComponent.java

```diff
@@ -0,0 +1,894 @@
+                                    updateSettingsIcon(severeReportFilter, "black", "black");                                     
+                                }
+                                if (errorReportLevels.contains("WARNING")) {
+                                    updateSettingsIcon(warnReportFilter, "black", "black");
+                                }
+                                if (errorReportLevels.contains("INFO")) {
+                                    updateSettingsIcon(infoReportFilter, "black", "black");
+                                }
+                                if (errorReportLevels.contains("FINE")) {
+                                    updateSettingsIcon(fineReportFilter, "black", "black");
+                                }
+                                
+                                if (errorPopupLevels.contains("SEVERE")) {
+                                    updateSettingsIcon(severePopupAllowed, "black", "black");
+                                }
+                                if (errorPopupLevels.contains("WARNING")) {
+                                    updateSettingsIcon(warnPopupAllowed, "black", "black");
+                                }
+                                if (errorPopupLevels.contains("INFO")) {
+                                    updateSettingsIcon(infoPopupAllowed, "black", "black");
+                                }
+                                if (errorPopupLevels.contains("FINE")) {
+                                    updateSettingsIcon(finePopupAllowed, "black", "black");
+                                }
+                            });
+                            Thread.sleep(650);
+                        } catch (final InvocationTargetException | InterruptedException ex) {
+                            Exceptions.printStackTrace(ex);
+                        }
+                    }
+                    updateSettings();
+                });
+                flasher.start();
+            } catch (final IOException ex) {
+                Exceptions.printStackTrace(ex);
+            }
+        }
+    }
+
+    public void removeEntry(final int indexPosition) {
+        sessionErrors.remove(indexPosition);
+        updateSessionErrorsBox(-1);
+    }
+
+    public int getPopupControlValue() {
+        return popupMode;
+    }
+
+    private ArrayList<String> getErrorLevelList(final boolean unAcknowledgedEntriesOnly) {
+        final ArrayList<String> resultList = new ArrayList<>();
+        for (ErrorReportEntry entry : sessionErrors) {
+            if (!resultList.contains(entry.getErrorLevel().getName())) {
+                if (unAcknowledgedEntriesOnly && entry.getLastDate().after(flashActivatedDate)) {
+                    resultList.add(entry.getErrorLevel().getName());
+                } else if (!unAcknowledgedEntriesOnly) {
+                    resultList.add(entry.getErrorLevel().getName());
+                }
```

Comment:
given the if and else if have the same subsequent flow, these should be combined